kernel/kernel-api/src/main/java/io/delta/kernel/coordinatedcommits/CommitCoordinatorUtils.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
We also have [CoordinatedCommitUtils.java](https://github.com/delta-io/delta/blob/master/storage/src/main/java/io/delta/storage/commit/CoordinatedCommitsUtils.java)

Would be nice to either merge them in the same file eventually or have better names for them

Reply:
Eventually, everything Coordinated-Commits-related in delta-storage will be refactored and moved to delta-kernel, and then be deleted.